### PR TITLE
Configure Deneb on Goerli

### PR DESCRIPTION
**Pull request by StefanBratanov**
## PR Description
Configuring Deneb on Goerli as per: https://github.com/eth-clients/goerli/pull/178

## Documentation

- [X] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [X] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/prater.yaml [hunk 1]
```diff
@@ -71,6 +74,8 @@ EJECTION_BALANCE: 16000000000
 MIN_PER_EPOCH_CHURN_LIMIT: 4
 # 2**16 (= 65,536)
 CHURN_LIMIT_QUOTIENT: 65536
+# [New in Deneb:EIP7514] 2**3 (= 8)
+MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 8
 
 
 # Fork choice
```
Comment (lucassaldanha) [inline ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/prater.yaml:81]: Are we missing updating PROPOSER_SCORE_BOOST to 40?

Comment (StefanBratanov) [reply ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/prater.yaml:81]: Yeah, think so. Changed it. But this is not DENEB specific, so it's weird it hasn't been changed.

File: ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/prater.yaml [hunk 2]
```diff
@@ -119,4 +124,16 @@ SUBNETS_PER_NODE: 2
...
+# `2**7` (=128)
+MAX_REQUEST_BLOCKS_DENEB: 128
+# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK
+MAX_REQUEST_BLOB_SIDECARS: 768
+# `2**12` (= 4096 epochs, ~18 days)
+MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: 4096
+# `6`
+BLOB_SIDECAR_SUBNET_COUNT: 6
+# `uint64(6)`
+MAX_BLOBS_PER_BLOCK: 6
```
Comment (tbenr) [inline ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/prater.yaml]: shouldn't be inherited from mainnet preset and not be specified here?

Comment (tbenr) [reply ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/prater.yaml]: (as per merged config in https://github.com/eth-clients/goerli/pull/178/files)

Comment (StefanBratanov) [reply ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/prater.yaml]: very good catch, fixed

**Review by tbenr — APPROVED** [review]
LGTM

**Pull request merged**